python/ray/node.py

```diff
@@ -801,8 +819,8 @@ def start_reaper_process(self):
                 process_info,
             ]
 
-    def start_redis(self):
-        """Start the Redis server."""
+    def start_or_configure_redis(self):
```

Comment:
we probably will split this function when we do clean up and only have configure logic here. do you want to add a TODO for this?

Reply:
Added to the callsite.